repository/kafka/docs/latest/configuration.md

```diff
@@ -44,27 +44,47 @@ Deploying without persistent storage isn't recommended for production usage.
 
 By default, the Kafka cluster will have the JMX Exporter enabled. You can check more information around how KUDO Kafka exposes metrics in [monitoring](./monitoring.md).
 
-##### Zookeeper PATH
+##### Zookeeper Configuration
 
-By default, the Kafka cluster will try to use a zookeeper path same as specified in `--instance` flag.
-You can get the instance name by `kubectl get instances` 
+KUDO Kafka requires a running ZooKeeper ensemble to perform its own internal accounting and persist cluster topology. And connectivity to the Zookeeper.
+You can install the KUDO Zookeeper or an alternate Zookeeper to use with KUDO Kafka. 
 
-To override the default zk path, use the `ZOOKEEPER_PATH` parameter. In next example the  `ZOOKEEPER_PATH` will be
-`/custom-path`
+###### Configuring the Zookeeper connection:
 
+You can configure KUDO Kafka to use Zookeeper using the KUDO Kafka parameter `ZOOKEEPER_URI`
```

Comment:
too much KUDO kafka in this sentence, I don't understand. Maybe the second usage of that words can be dropped?

Reply:
I introduced this because initially we had some confusion that `ZOOKEEPER_URI` is something that configures KUDO Zookeeper